app/api/attendees.py

```diff
@@ -25,9 +26,23 @@
 from app.models.ticket_holder import TicketHolder
 from app.models.user import User
 
+from app.settings import get_settings
+
 attendee_misc_routes = Blueprint('attendee_misc', __name__, url_prefix='/v1')
 
 
+def get_ticket_counts(event_id):
+    order_expiry_time = get_settings()['order_expiry_time']
+    return db.session.query(TicketHolder.id).join(Order).filter(TicketHolder.order_id == Order.id) \
+        .filter(Order.event_id == int(event_id),
+                Order.deleted_at.is_(None),
+                or_(Order.status == 'placed',
+                    Order.status == 'completed',
+                    and_(Order.status == 'initializing',
```

Comment:
@iamareebjamal what about pending orders then? This condition will fail to count them.

Reply:
How?

Reply:
For pending order we give a window for 30 mins to complete payment.

Reply:
So additional 30 minutes is granted to attendee inspite of order_expiry_time,which is not taken in account here.